CMakeLists.txt

```diff
@@ -77,19 +77,23 @@ add_dependencies(show_sphere ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORT
 target_link_libraries(show_sphere ${catkin_LIBRARIES})
 
 add_executable(show_sphere_and_arrow src/show_sphere_and_arrow.cpp)
+add_dependencies(show_sphere ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
```

Comment:
targets are not right for some of these.  Need to set the target name to the executable name for each.  i.e. "show_sphere_and_arrow", "show_sphere_and_arrow_color".